capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative('./decorator')
+
+class CapitalizeDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] [REQUIRED] Good job so far! Kindly change `.upcase` ❌ to `.capitalize` ✔️ because upcase implement `UPPERCASE` all elements `Captalize` implement start with capital leter, please implement it 🙏🏼

Reply:
Ok, thanks.